### PR TITLE
Create automated build and test pipeline

### PR DESCRIPTION
**Pull request by aleks-ivanov**
- add all necessary steps to initiate a dotnet build process and a dotnet test process as per BUILDING.md

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,17 +1,58 @@
...
+        id: build_solution
+        shell: pwsh
+        run: |
+          msbuild .\iTextCore.netstandard.sln
+
+      - name: Run tests
+        id: run_tests
+        shell: pwsh
+        run: |
+          $env:ITEXT_GS_EXEC = "C:\Program Files\gs\gs9.53.3\bin\gswin64c.exe"
```
Comment (kulov) [inline .github/workflows/main.yml:54]: Why do we need these environment variables to be set?

**Review by kulov — CHANGES_REQUESTED** [review]
Why do we have failing tests?

File: .github/workflows/main.yml (hunk 1 above)
Comment (aleks-ivanov) [reply .github/workflows/main.yml:54]: @kulov the build process executes both testing software directly, so it uses these two variables which should be paths to .exe file as per their BUILDING.md

**Comment by aleks-ivanov** [comment]
> Why do we have failing tests?

@kulov Different tests are failing at different runs both locally and on GH.
I thought to submit the PR and communicate with the owner about it, if it's something we are doing wrong and can fix.

**Pull request merged**